scripts/get-gee-build-environment.sh

```diff
@@ -0,0 +1,432 @@
+#! /bin/bash
```

Comment:
We should update the build documentation to give people the option to use this file instead of installing everything by hand.

Reply:
So essentially just a preliminary "are you sure you want to do this?" option?

Reply:
I was thinking of a change to earth_enterprise/BUILD.md - it basically has instructions for how to do this by hand and it would be a good idea to point users here as another option.

Reply:
Change shebang line to the more portable `#!/usr/bin/env bash`

Reply:
If there is no bash specific requirements in this script, then really it probably should just be #!/bin/sh